packages/assets-controllers/src/Standards/NftStandards/ERC1155/ERC1155Standard.ts

```diff
@@ -161,13 +209,31 @@ export class ERC1155Standard {
     standard: string;
     tokenURI: string | undefined;
     image: string | undefined;
+    name: string | undefined;
+    symbol: string | undefined;
   }> => {
     const isERC1155 = await this.contractSupportsBase1155Interface(address);
 
     if (!isERC1155) {
       throw new Error("This isn't a valid ERC1155 contract");
     }
-    let tokenURI, image;
+
+    let tokenURI, image, symbol, name;
+
+    const results = await Promise.allSettled([
+      this.getAssetName(address),
+      this.getAssetSymbol(address),
+    ]);
+
+    const fulfilledResults = results.filter(
+      (result): result is PromiseFulfilledResult<string> =>
+        result.status === 'fulfilled',
+    );
+
+    if (fulfilledResults.length > 0) {
+      name = fulfilledResults[0].value;
```

Comment:
If `getAssetName` fails but `getAssetSymbol` succeeds, wouldn't this be the wrong index?  The 0th fuffilled promise would represent `symbol`.